site/th.njk

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+meta_title: การเคลื่อนไหวเพื่อหยุดยั้งโรคระบาด COVID-19
+meta_description: รัฐบาลไม่สามารถจัดการได้ฝ่ายเดียว เราต้องเริ่มเดี๋ยวนี้ ด้วยการทำหน้าที่ของเรา แล้วอยู่บ้านซะ
+body_class: home
+title: '#StayTheFuckHome'
+lang: en
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">การเคลื่อนไหวเพื่อหยุดยั้งโรคระบาด COVID-19</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+                รัฐบาลของพวกเรากำลังดำเนินการเพื่อที่จะควบคุมการระบาดของโรค
+                <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2">SARS-CoV-2</a>อย่างเชื่องช้า
+                ทำให้เกิด <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019">การระบาด COVID-19 เป็นวงกว้าง</a> ขึ้น
+                ด้วยการตอบสนองที่ไม่ทันต่อสถานการณ์ นโยบายที่ทำได้แค่จำยอมต่อสถานการณ์ และความพยายามที่จะประคองเศรษฐกิจที่ทำให้
+                ไม่สามารถใช้มาตรการที่จะปกป้องผู้คนนับล้านจากโรคระบาด อย่างไรก็ตาม มันไม่ใช่แค่หน้าที่ของรัฐบาลที่จะจัดการกับวิกฤตครั้งนี้
+                มันเป็นหน้าที่ของพวกเรา ในฐานะประชากรโลก ที่จะต้องทำตอนนี้เพื่อที่จะต่อกรกับโรคระบาด COVID-19
+                <br /><br />
+                Let's put it bluntly: <strong>Stay The Fuck Home!</strong>
```

Comment:
You probably just forgot this; might want to say something like: หยุด! ช่วยอยู่บ้าน